### PR TITLE
Remove redundant body end tag and modify the dependency installation

### PR DESCRIPTION
**Pull request by bbiiggppiigg**
Remove redundant body end tag in problem.html
Modify dependency installation of django
(pip install django will install django1.8 and it is incompatible with the code)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install https://www.djangoproject.com/download/1.7.8/tarball/
```
Comment (bruce3557) [inline README.md]: Should we specify the version of django??

Comment (bbiiggppiigg) [inline README.md]: I tried to install the whole thing yesterday, and it turns out that the default django installation of version 1.8 is incompatible with some of our existing codes. Though it is easy to fix the code to match newer version, I wonder should we keep the code compatible to the newest versions of our dependencies throughout the whole implementation process or should we stick to a particular version.

Comment (nickelcheng) [inline README.md]: Sorry, I had encountered that trouble but forgot to modify readme.
pip install django==1.7 is enough

File: README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install django==1.7
```
Comment (bruce3557) [inline README.md]: still confuse about django version
why not > 1.7
Any reason we need to fix the version?

Comment (nickelcheng) [inline README.md]: It's django's [bug](https://code.djangoproject.com/query?owner=nobody&desc=1&order=id&row=description)

**Comment by bruce3557** [comment]
I think this is ok
So if you need it, please reopen this.

**Pull request closed without merging**